contrib/lkt/language/parser.py

```diff
@@ -2449,6 +2460,23 @@ def check_correctness_pre():
                     )
                 )
             )
+            # In case of PropertyError, check for RaiseExpr
+            ._or(
+                If(
+                    And(
+                        rd.result_ref == Entity.property_error_type,
+                        Not(Entity.first_no_paren_parent.is_a(T.RaiseExpr))
+                    ),
+
+                    Self.error(
+                        S("cannot call ").concat(
+                            Entity.property_error_type.full_name
+                        ).concat(S(" without a raise expression"))
```

Comment:
To be more specific, I would reword: “cannot call … outside of a raise expression”.